pymemcache/test/test_utils.py

```diff
@@ -110,4 +120,4 @@ def test_prepand_append():
     client.set(b"k", "1")
     client.append(b"k", "a")
     client.prepend(b"k", "p")
-    assert client.get(b"k") == b"p1a"
+    assert client.get(b"k") == "p1a"
```

Comment:
I'm not sure this test is correct. Without `pickle_serde`, it really must be bytes.

Reply:
I reverted this change and implemented a fix for the right test.